linux_os/guide/services/base/service_abrtd_disabled/rule.yml

```diff
@@ -28,7 +28,7 @@ references:
     stigid@rhel6: "000261"
     srg@rhel6: SRG-OS-000096
     disa@rhel6: '381'
-    nist: AC-17(8),CM-7
+    nist: CM-6(a)
```

Comment:
@shawndwells should this be `CM-7(a),CM-7(b),CM-6(a)` ?

Reply:
Earlier review had this as ``CM-7(a),CM-6(a)``

Reply:
@shawndwells ya... think this was a typo since we discussed that for disabling `CM-7(a),CM-7(b),CM-6(a)` should be the default.